README.md

```diff
@@ -49,15 +118,349 @@ src/
+- Unordered lists
+  - Unordered lists with nesting
+    - Unordered lists with second-level nesting
+    - Unordered lists with second-level nesting
+- Unordered lists #2
+  - Unordered lists with nesting #2
+- Unordered lists #3
+- Unordered lists #4
+  - Unordered lists with nesting #4
+  - Unordered lists with nesting #4
+```
+
+### Ordered List
+
+<img src="./readme-imgs/ordered-list.png" alt="Ordered List" width="600"/>
+
+```
+1. [`Set Options`](../api/resources/operations/object/set-options.mdx) to set the flags on the issuing account to `0x1` to enable `AUTHORIZATION REQUIRED`. This is necessary because you cannot run the [`Allow Trust`](../api/resources/operations/object/allow-trust.mdx) operation without `AUTHORIZATION REQUIRED` being set on your issuing account.
+1. [`Allow Trust`](../api/resources/operations/object/allow-trust.mdx) on the existing user's account in order to authorize it. **Note:** You can actually place as many as `MAX OPERATIONS PER TRANSACTION - 2` (currently the [maximum is 100](concepts/transactions.md#list-of-operations)) `Allow Trust` operations for different accounts to minimize the number of transactions submitted to the network.
+1. [`Set Options`](../api/resources/operations/object/set-options.mdx) to set the flags on the issuing account to `0x0` to disable `AUTHORIZATION REQUIRED`.
+   1. If I had more points to make about `Set Options`, I'd start listing those here.
+   1. Another point
+      1. Nested point of Anoter point
+      1. Nested point of Anoter point
+      1. Nested point of Anoter point
+   1. Another point #2
+```
+
+### Images
+
+Currently, only [Documentation](https://developers.stellar.org/docs/) uses
+images.
+
+1. Drop the images in the `content/docs/web-assets` folder
+2. Refer to the asset as `![image caption](web-assets/image-file-name.png)` in
+   the proper location in your file - make sure that `web-assets`' relative path
+   is correct
+3. Captions: Include alt text if you’d like a caption:
+   `![This is the image caption](web-assets/image-file-name.png)` Leave the alt
+   text section blank if you do not want a caption:
+   `![](web-assets/image-file-name.png)`
+
+### Quote
+
+![Quote](./readme-imgs/quote.png)
+
+```
+> Here's a quote
+```
+
+### Paragraph and Headings
+
+Beyond defining title font sizes, line heights, and weights:
+
+- **H1** tags are reserved for the page’s title and should not be used; that
+  said, if they are used, they will still show up on the front end as H1 tags
+- **H2** tags populate the ride-side page
+  [table of contents](https://developers.stellar.org/docs/enabling-deposit-and-withdrawal/setting-up-test-server/)
+  in Documentation
+
+## Custom Components
+
+Custom (React) components that are being used throughout Documentation and API
+Reference.
+
+### AttributeTable
+
+<img src="./readme-imgs/attribute-table.png" alt="AttributeTable Component" width="600"/>
+
+`<AttributeTable/>` is currently used in API Reference. Nested Table is allowed.
```

Comment:
I'm not sure these "X is currently used on Y" is helpful information for content editors, and it seems likely to get out of date. I don't feel strongly but maybe these lines should be removed, thoughts?

I'm not sure exactly what "Nested Table is allowed" is meant to convey, maybe this should explain how nested attributes turn into "child attributes" that are collapsed by default?

Reply:
Good call. I will mention "X is currently used on Y" for `<ExampleResponse/>` that is being used only in API Reference. I will add a better explanation for the latter.

Reply:
I think I want to imply where each component is being used so a reader can find it easier if he/she wants to see it in action (but at the same time, I added a link..)

Reply:
Removed!